LICENSE

```diff
@@ -1,6 +1,6 @@
                                  Apache License
                            Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+                        https://www.apache.org/licenses/
```

Comment:
This was from https://www.apache.org/licenses/LICENSE-2.0, we should not change here.

Reply:
Done